app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
```

Comment:
Validate the lenght of command - it should be exactly 3 arguments

And then use unpacking to move command parts to variables
command, source_file, target_file = command